i18n/en-US.properties

```diff
@@ -938,6 +938,12 @@ boxui.collaboratorAvatars.viewAdditionalPeopleText = View additional people
 boxui.collapsiblesidebar.collapseBtnLabel = Collapse
 # Aria label for toggle button that expands/collapses sidebar (collapsed state)
 boxui.collapsiblesidebar.expandBtnLabel = Expand
+# Content Answers submit input button text
+boxui.contentAnswers.ask = Ask
```

Comment:
It looks like something went awry during the pre-commit hook.

Reply:
Yep, it amended last commit for me and said to push with --no-verify, is this okay? Or should I revert changes for this file?

Reply:
It was pre-push hook

Reply:
Yeah, I would revert. Maybe @greg-in-a-box can comment on the automation here.

Reply:
Reverted them, lint_test_build was failing on them https://github.com/box/box-ui-elements/runs/31305517831